website/docs/getting-started.md

```diff
@@ -11,6 +11,29 @@ import {HeaderBadgesWidget} from '@site/src/components/HeaderBadgesWidget.js';
 
 [Prysm](https://github.com/prysmaticlabs/prysm) is an [Ethereum](https://ethereum.org/en/developers/docs/intro-to-ethereum/) [proof-of-stake](https://ethereum.org/en/developers/docs/consensus-mechanisms/pos/) client written in [Go](https://golang.org). You can use Prysm to participate in Ethereum's [decentralized economy](https://ethereum.org/en/developers/docs/web2-vs-web3/) by [running a node](./install/install-with-script.md) and, if you have [32 ETH to stake](https://ethereum.org/en/staking/), a [validator client](./install/install-with-script.md#step-6-run-a-validator-using-prysm). If you're new to Ethereum, you may enjoy our beginner-friendly [Nodes and networks](./concepts/nodes-networks.md) explainer.
 
+:::info
+
+## Upcoming Hardfork: Deneb/Cancun Q1 2024 (February)
+
+Requirements: Upgraded Prysm (validator & beacon node) / Upgraded Execution client
+New Data Storage requirement from [EIP-4844](https://eips.ethereum.org/EIPS/eip-4844): 52GB ~ 104GB storage requirement increase caused by new blob retention requirements. read more [here](./concepts/blobs.md)
```

Comment:
![Screenshot_2024-01-31_10-38-56](https://github.com/prysmaticlabs/documentation/assets/7246818/fff162fe-4f62-4fde-b0ea-153f1ba08428)

Formatting of this looks weird. Please check punctuation, capitalization, etc